website/static/index.html

```diff
@@ -1,31 +1,132 @@
-<!DOCTYPE HTML>
-<html lang="en-US">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="refresh" content="0; url=docs/overview">
-        <link rel="apple-touch-icon" sizes="57x57" href="/img/icons/apple-icon-57x57.png">
-        <link rel="apple-touch-icon" sizes="60x60" href="/img/icons/apple-icon-60x60.png">
-        <link rel="apple-touch-icon" sizes="72x72" href="/img/icons/apple-icon-72x72.png">
-        <link rel="apple-touch-icon" sizes="76x76" href="/img/icons/apple-icon-76x76.png">
-        <link rel="apple-touch-icon" sizes="114x114" href="/img/icons/apple-icon-114x114.png">
-        <link rel="apple-touch-icon" sizes="120x120" href="/img/icons/apple-icon-120x120.png">
-        <link rel="apple-touch-icon" sizes="144x144" href="/img/icons/apple-icon-144x144.png">
-        <link rel="apple-touch-icon" sizes="152x152" href="/img/icons/apple-icon-152x152.png">
-        <link rel="apple-touch-icon" sizes="180x180" href="/img/icons/apple-icon-180x180.png">
-        <link rel="icon" type="image/png" sizes="192x192"  href="/img/icons/android-icon-192x192.png">
-        <link rel="icon" type="image/png" sizes="32x32" href="/img/icons/favicon-32x32.png">
-        <link rel="icon" type="image/png" sizes="96x96" href="/img/icons/favicon-96x96.png">
-        <link rel="icon" type="image/png" sizes="16x16" href="/img/icons/favicon-16x16.png">
-        <link rel="manifest" href="/manifest.json">
-        <meta name="msapplication-TileColor" content="#ffffff">
-        <meta name="msapplication-TileImage" content="/img/icons/ms-icon-144x144.png">
-        <meta name="theme-color" content="#ffffff">
-        <script type="text/javascript">
-            window.location.href = "docs/overview"
-        </script>
-        <title>All Contributors: ✨ Recognize all contributors, not just the ones who push code ✨</title>
-    </head>
-    <body>
-        If you are not redirected automatically, follow this <a href='docs/overview'>link</a>.
-    </body>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins:300,400,400i,600"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/custom.css" />
+    <title>All Contributors</title>
+  </head>
+  <body>
+    <header class="container">
+      <img src="https://source.unsplash.com/350x40" alt="logo" />
+      <nav>
+        <ul>
+          <li>Docs</li>
+          <li>About</li>
+          <li>Github</li>
+        </ul>
+      </nav>
+    </header>
+    <div class="hero container">
+      <div class="col-6">
+        <h1>Recognize All Contributors</h1>
+        <p>Including those that don't push code</p>
+        <button class="hero-button">Add to Project</button>
```

Comment:
i'd use `<a>` here, especially considering this will be a link

Way fewer styles for you to override.